car-repair-html5-bootstrap-template/readme.txt

```diff
@@ -21,7 +24,7 @@ License :
 
 - You are allowed to use all files for both personal and commercial projects.
 
-- If you use/modify the resources in your projects,we�d appreciate a linkback to this site.
+- If you use/modify the resources in your projects,we’d appreciate a linkback to this site.
```

Comment:
This would be better
`If you use/modify the resources in your projects, we\'d appreciate a linkback to this site.`